### PR TITLE
Add choices when selecting credential Subject in CLI

### PR DESCRIPTION
**Pull request by tientaidev**
Adding choices when select Subject DID instead of always default it to the first identifier. 
This PR fix issues mentioned in #897

### CODE REVIEW

**Review by simonas-notcat — CHANGES_REQUESTED** [review]
@tientaidev , thanks for your contribution! I would suggest using `type: 'autocomplete'`, so that it's possible to select a DID from a list of known DIDs, but also to manually input any DID. Take a look at how it is done in `sdr` flow:

https://github.com/uport-project/veramo/blob/next/packages/cli/src/sdr.ts#L39

File: packages/cli/src/credential.ts [hunk 1]
```diff
@@ -39,12 +45,24 @@ credential
...
+          const res = fuzzy
+            .filter(input, subjects)
+            .map((el: any) => (typeof el === 'string' ? el : el.original))
+          return res
+        },
+        validate: (val) => {
+          if (val && typeof val !== 'string') {
+            val = val.value
+          }
+          if (!val || !val.startsWith('did:')) {
```
Comment (mirceanis) [inline packages/cli/src/credential.ts:61]: A subject can be a plain URL, it doesn't have to start with `did:`

It is even OK if the subject is missing, in which case the `credentialSubject.id` property of the credential should be omitted.

**Review by mirceanis — CHANGES_REQUESTED** [review]
Thank you for the contribution!

The validation is a bit too strict, but otherwise it looks good.

File: packages/cli/src/credential.ts (hunk 1 above)
Comment (mirceanis) [reply packages/cli/src/credential.ts:61]: if using anything other than a DID causes issues in other places, we can merge this as is and fix the other places later, but please flag it if that is the case

**Review by mirceanis — APPROVED** [review]
Looks great,
Thank you!

**Pull request merged**